gson/src/main/java/com/google/gson/internal/$Gson$Types.java

```diff
@@ -294,6 +294,22 @@ public static Type getArrayComponentType(Type array) {
         : ((Class<?>) array).getComponentType();
   }
 
+  /**
+   * Returns the element type of this iterable type.
+   * @throws IllegalArgumentException if this type is not an iterable.
+   */
+  public static Type getIterableElementType(Type context, Class<?> contextRawType) {
+    Type collectionType = getSupertype(context, contextRawType, Iterable.class);
```

Comment:
Change `collectionType` to `iterableType`?

Reply:
Yes, thanks.